### PR TITLE
Restyled the Status Bar page

### PR DESCRIPTION
**Pull request by System-x64**
The info of 3 separate bars with different config styles displayed back to back made it look very convoluted so I decided to separate all their info into different tabs just like we have in the _Installation_ page.

Also if y'all like this layout...shall I_re-lay_ the App Clients page like this aswell?

### CODE REVIEW

File: pages/Useful Utilities/Status-Bars.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Waybar
+{{< tabs "uniqueid" >}}
```
Comment (fufexan) [inline pages/Useful Utilities/Status-Bars.md:1]:
```suggestion
Status bars on Hyprland.

{{< tabs "uniqueid" >}}
```

Apparently hugo won't build the page if `{` is the first thing it sees. Use a different sentence if you find a better one.

File: pages/Useful Utilities/Status-Bars.md [hunk 2]
```diff
@@ -73,12 +79,16 @@ Read
...
 
-You can install it from the AUR by the name `hybrid-bar`.
+You can install it from the AUR by the name `hybrid-bar` or `hybrid-bar-git`.
 
 ## Configuration
 
 The configuration is done through JSON, more information is available [here](https://github.com/vars1ty/HybridBar).
+
+{{</ tab >}}
+{{< /tabs >}}
```
Comment (fufexan) [inline pages/Useful Utilities/Status-Bars.md:94]:
```suggestion
{{</ tabs >}}
```

**Review by fufexan — COMMENTED** [review]
Generally looks fine to me. @vaxerski thoughts?

**Comment by vaxerski** [comment]
No clue why, but fine

**Pull request closed without merging**